shim.js

```diff
@@ -43,3 +43,7 @@ if (typeof atob === 'undefined') {
     return Buffer.from(b64Encoded, 'base64').toString('binary');
   };
 }
+
+const { URLSearchParams } = require('react-native-url-polyfill');
+
+global.URLSearchParams = URLSearchParams;
```

Comment:
Do we use this in this repo?

Reply:
No, but reown uses it, crashes without this